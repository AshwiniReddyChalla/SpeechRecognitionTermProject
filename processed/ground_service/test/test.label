ground_service
ground_service
ground_service
ground_service
ground_service
ground_service
ground_service
ground_service
ground_service
ground_service
ground_service
ground_service
ground_service
ground_service
ground_service
ground_service
ground_service
ground_service
ground_service
ground_service
ground_service
ground_service
ground_service
ground_service
ground_service
ground_service
ground_service
ground_service
ground_service
ground_service
ground_service
ground_service
ground_service
ground_service
ground_service
ground_service
