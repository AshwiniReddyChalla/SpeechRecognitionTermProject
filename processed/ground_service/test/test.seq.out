O B-airport_name I-airport_name O O O O O O O O O
O O O O O O O B-airport_name I-airport_name B-city_name I-city_name O O B-city_name
O O O O O B-city_name
O O O O O B-city_name I-city_name
O O O O O O B-city_name O B-day_name B-period_of_day
O O O O O O O O O O O B-city_name B-period_of_day B-period_of_day
O O O O O B-transport_type O B-city_name B-state_code
O O O O O B-transport_type O B-city_name
O O O O O O B-city_name
O O O O O O B-airport_name I-airport_name I-airport_name I-airport_name O O B-city_name
O O O O O O B-airport_name I-airport_name O O B-city_name
O O O O O O B-airport_name I-airport_name O O B-city_name
O O O O O O B-airport_code O O B-city_name
O O O O O O B-airport_name I-airport_name O O B-city_name
O O O O O B-city_name
O O O O O O O O B-city_name
O O O O O B-city_name I-city_name
O O O O O O O B-airport_name I-airport_name
O O O O O O O B-airport_name I-airport_name
O O O O O O O B-airport_name I-airport_name I-airport_name I-airport_name
O O O O O O O B-airport_name I-airport_name
O O O O O B-transport_type O B-city_name
O O O B-city_name I-city_name
O O O B-city_name I-city_name
O O O O O B-city_name
O O O O O B-city_name I-city_name
O O O O O O B-city_name
O O O O B-city_name
O O O O O O B-airport_name I-airport_name O O O O O O O O O O B-period_of_day
O O O O O O B-airport_name I-airport_name I-airport_name O B-city_name I-city_name I-city_name
O O O O O B-airport_code O B-city_name I-city_name I-city_name
O O O O O O B-airport_code O B-city_name I-city_name I-city_name
O O O O O B-airport_code O B-city_name I-city_name I-city_name
O O O O O O O B-airport_name I-airport_name
O O O O O O O O O B-airport_name I-airport_name I-airport_name
O O B-transport_type I-transport_type O O B-airport_name I-airport_name
